src/odemis/gui/comp/overlay/base.py

```diff
@@ -1107,6 +1572,7 @@ def __init__(self, colour=gui.SELECTION_COLOUR, center=(0, 0), edit_mode=EDIT_MO
         self.edit_v_start_pos = None  # The view port coordinates where a drag/edit originated
         self.edit_hover = None  # What edge is being edited (gui.HOVER_*)
         self.edit_mode = edit_mode
+        self.edit_v_point_idx = None
```

Comment:
Please add type and short description in comment (something like "int, index of the edge being moved")